frontend/common/utils/utils.tsx

```diff
@@ -294,7 +268,7 @@ const Utils = Object.assign({}, require('./base/_utils'), {
         break
       }
       case 'AUTO_SEATS': {
-        valid = isScaleupOrGreater && Utils.getFlagsmithHasFeature('auto_seats')
+        valid = isScaleupOrGreater && !isEnterprise
```

Comment:
Interestingly, this is logically equivalent to `isScaleUp` because there's nothing above scale up except enterprise, but maybe this allows us to handle additional plans in the future (although I'm not sure we will need to).